packages/eas-cli/src/update/__tests__/update-test.ts

```diff
@@ -300,4 +302,26 @@ describe('UpdatePublish', () => {
       ]);
     });
   });
+
+  describe(truncatePublishUpdateMessage.name, () => {
+    it.each([Array.from({ length: 2024 }, () => 'a').join(''), 'Small message =)'])(
+      'truncates messages to a length of 1024, including ellipses',
+      message => {
+        const warnSpy = jest.spyOn(Log, 'warn');
+        const truncatedMessage = truncatePublishUpdateMessage(message);
+        if (message.length > 1024) {
+          expect(truncatedMessage.length).toEqual(1024);
+          expect(truncatedMessage.slice(-3)).toEqual('...');
+          expect(warnSpy).toBeCalledWith(
+            'Update message exceeds the allowed charater limit. Truncating message...'
+          );
+          warnSpy.mockClear();
+        } else {
+          expect(truncatedMessage.length).toEqual(message.length);
+          expect(truncatedMessage.slice(-3)).toEqual(message.slice(-3));
```

Comment:
Isn't this the same as `truncatedMessage.toEqual(message)`?

Reply:
it is a worse version of `truncatedMessage.toEqual(message)` 😄